cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/aggregator/KafkaPartitionMetricSampleAggregatorTest.java

```diff
@@ -78,11 +78,14 @@ public void testRecentSnapshot() throws NotEnoughValidWindowsException {
     for (int i = 0; i < NUM_WINDOWS; i++) {
       assertEquals((NUM_WINDOWS - i) * WINDOW_MS, result.valuesAndExtrapolations().get(PE).window(i));
       for (Resource resource : Resource.values()) {
-        double expectedValue = resource == Resource.DISK ?
+        Collection<Integer> metricIds = KafkaMetricDef.resourceToMetricIds(resource);
+        double expectedValue = (resource == Resource.DISK ?
             (NUM_WINDOWS - 1 - i) * 10 + MIN_SAMPLES_PER_WINDOW - 1 :
-            (NUM_WINDOWS - 1 - i) * 10 + (MIN_SAMPLES_PER_WINDOW - 1) / 2.0;
+            (NUM_WINDOWS - 1 - i) * 10 + (MIN_SAMPLES_PER_WINDOW - 1) / 2.0) * metricIds.size();
```

Comment:
It is not clear why we multiply with number of metrics here.

Reply:
Explained above, basically it was because we have more than one metrics for the same resource now. Each metric has the same values, so we need to multiply by the number of metrics for the resource.